recipe/meta.yaml

```diff
@@ -19,31 +19,37 @@ outputs:
     script: build_base.bat  # [win]
     script: build_base.sh  # [not win]
     build:
-      skip: true  # [py<37]
+      skip: true  # [py<38]
       ignore_run_exports:
         - tk  # [linux]
+        - zlib
     requirements:
       build:
         - pkg-config  # [not win]
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
       host:
         - python
-        - pip
+        - pip >=9.0.1
```

Comment:
pip 9.0.1 is so old s you can remove pinning. cond-build always takes the latest version from host. You can make an experiment and check host logs.

Reply:
Removed pinning. Thanks!